03-rnaseq/dimension-reduction_rnaseq_01_pca.Rmd

```diff
@@ -238,49 +235,53 @@ Now we are going to use a combination of functions from the `DESeq2` and `ggplot
+      levels = c("pre-adt", "post-adt")
+    ),
     refinebio_disease = as.factor(refinebio_disease)
   )
 ```
 
 ## Create a DESeqDataset
 
 We will be using the `DESeq2` package for [normalizing and transforming our data](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#deseq2-transformation-methods), which requires us to format our data into a `DESeqDataSet` object.
-We turn the data frame (or matrix) into a [`DESeqDataSet` object](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#02_About_DESeq2). ) and specify which variable labels our experimental groups using the [`design` argument](http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#multi-factor-designs) [@Love2014].
+We turn the data frame (or matrix) into a [`DESeqDataSet` object](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html#02_About_DESeq2) and specify which variable labels our experimental groups using the [`design` argument](http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#multi-factor-designs) [@Love2014].
 In this chunk of code, we will not provide a specific model to the `design` argument because we are not performing a differential expression analysis.
 
 ```{r}
 # Create a `DESeqDataSet` object
 dds <- DESeqDataSetFromMatrix(
-  countData = df, # This is the data frame with the counts values for all replicates in our dataset
-  colData = metadata, # This is the data frame with the annotation data for the replicates in the counts data frame
-  design = ~1 # Here we are not specifying a model -- Replace with an appropriate design variable for your analysis
+  countData = df, # the counts values for all samples in our dataset
+  colData = metadata, # annotation data for the samples in the counts data frame
+  design = ~1 # Here we are not specifying a model
+  # Replace with an appropriate design variable for your analysis
 )
 ```
 
 ## Define a minimum counts cutoff
 
-We want to filter out the genes that have not been expressed or that have low expression counts because we want to remove any possible noise from our data before we normalize the data and create our plot.
+We want to filter out the genes that have not been expressed or that have low expression counts because we want to reduce noise in our data before we normalize  and create our plot.
 We are going to do some pre-filtering to keep only genes with 10 or more reads total.
+This threshold might vary depending on the number of samples and expression patterns acrosss your data set.
 Note that rows represent gene data and the columns represent sample data in our dataset.
 
 ```{r}
-# Define a minimum counts cutoff and filter `DESeqDataSet` object to include
-# only rows that have counts above the cutoff
+# Filter `DESeqDataSet` object to include only genes
+# with a total count of at least 10 reads across the samples
```

Comment:
Do you wanna squeeze in the #417 change while we are at it?

Reply:
I think I'd rather leave those for a separate PR.

Reply:
Okey dokey.